### PR TITLE
adding event handler names to avoid crash

### PR DESCRIPTION
**Pull request by thakurajayL**

### CODE REVIEW

File: common/events.go [hunk 1]
```diff
@@ -115,7 +122,7 @@ const (
 	SEC_MOD_COMPLETE_EVENT
 	SEC_MOD_REJECT_EVENT // 95
 
-	FIVE_GMM_STATUS_EVENT = UE_5GS_MOBILITY_MANAGEMENT_EVENTS + 14 + iota // 86
+	FIVE_GMM_STATUS_EVENT = 100 + iota // 100
```
Comment (gab-arrobo) [inline common/events.go:125]: The following minor change will fix the linter issue
```suggestion
	FIVE_GMM_STATUS_EVENT = 100 + iota // 100
```

**Review by yoursunny — APPROVED** [review]
I just saw this.
This is effective in avoiding #125 crash.
It prints this message and then the profile continues.

```text
2024-06-26T02:14:23Z [INFO][GNBSIM][SimUe][imsi-001017005551000] Handling event: CONFIG-UPDATE-COMMAND
2024-06-26T02:14:23Z [WARN][GNBSIM][SimUe][imsi-001017005551000] Event: CONFIG-UPDATE-COMMAND is not supported
```

**Review by gab-arrobo — APPROVED** [review]
+1

**Comment by gab-arrobo** [comment]
close #125

**Pull request merged**